crates/arroyo-connectors/src/iggy/profile.json

```diff
@@ -0,0 +1,63 @@
+{
+    "type": "object",
+    "title": "IggyConfig",
+    "properties": {
+        "endpoint": {
+            "type": "string",
+            "title": "Endpoint",
+            "description": "The Apache Iggy server endpoint, only HTTP is currently supported",
+            "examples": [
+                "localhost:8090"
+            ]
+        },
+        "transport": {
+            "type": "string",
+            "title": "Transport Protocol",
+            "description": "The transport protocol to use for connecting to Iggy, only HTTP is currently supported",
+            "enum": [
+                "http"
+            ],
+            "default": "http"
+        },
+        "authentication": {
+            "type": "object",
+            "oneOf": [
+                {
+                    "type": "object",
+                    "title": "None",
+                    "properties": {},
+                    "additionalProperties": false
+                },
+                {
+                    "type": "object",
+                    "title": "Username/Password",
+                    "properties": {
+                        "username": {
+                            "title": "Username",
+                            "type": "string",
+                            "description": "Username for authentication"
```

Comment:
It's useful for username to also be a var-str, even though it's not generally sensitive, as that allows users to manage usernames and passwords together.